python/mirascope/llm/clients/anthropic/_utils/encode.py

```diff
@@ -23,6 +23,19 @@
 
 DEFAULT_MAX_TOKENS = 16000
 
+AnthropicImageMimeType = Literal["image/jpeg", "image/png", "image/gif", "image/webp"]
+
+
+def encode_image_mime_type(
```

Comment:
This function name is a bit confusing. It does not encode or convert. Could you suggest a clearer name that reflects validation?